platform/linux-generic/test/scripts/l2fwd_simple/pktio_env

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+#
+# Copyright(C) 2020 Marvell.
```

Comment:
A minor issue but it's better to keep the copyright message format consistent across the whole project.
So `# Copyright (c) 2020, Marvell`

Reply:
Ack.